apps/trading/components/market-selector/asset-dropdown.tsx

```diff
@@ -33,7 +33,7 @@ export const AssetDropdown = ({
       }
     >
       <DropdownMenuContent>
-        {assets?.map((a) => {
+        {assets?.filter(Boolean).map((a) => {
```

Comment:
We should need the optional chain nor the filter? We check for !assets?.length above?